src/compile/selection/toggle.ts

```diff
@@ -9,11 +9,18 @@ const toggle: SelectionCompiler<'point'> = {
   },
 
   signals: (model, selCmpt, signals) => {
-    return signals.concat({
-      name: selCmpt.name + TOGGLE,
-      value: false,
-      on: [{events: selCmpt.events, update: selCmpt.toggle}]
-    });
+    if (isTimerSelection(selCmpt)) {
```

Comment:
If toggle is always set to `false` in the case of timer selections, it might be preferable to augment `defined` above? i.e., saying that `toggle` can never be instantiated on a timer selection—so the resultant signals/etc. are all bypassed, and has the knock on effects throughout the selection codebase.

Reply:
That makes sense. Is there documentation for the semantics of `defined`? If not I can try to add it in this or a follow-up PR.

Reply:
Actually maybe I'll defer to jz on this since it's `false` in both branches. I'm not 100% sure what the behavior of this is supposed to be.

Reply:
using defined feels like the right thing, thanks: https://github.com/vega/vega-lite/pull/8921/commits/cac66c9bab5d35e103da1712b2e8c8180a4b4436